### DISCUSSION TITLE
12Pヘッダーの付け直しについて

### DISCUSSION THREAD

**Discussion by Piroshki01** [Yowkees/keyball#307] [Q&A] [answered]
誤って12Pヘッダの位置を間違えてPCBとProMicroをはんだ付けしてしまいました。
ハンダ吸い取り線でPCB側のハンダを極力吸ってみたのですが、完全に吸い取れず、外すのに苦慮しています。

電子工作初心者なので、修復する方法をご教授願います。

![16920273070626919197557900180974](https://github.com/Yowkees/keyball/assets/142245946/7250f3a9-eb3c-469a-9b95-86aa4e94b3dd)

![16920273953363532486574363085733](https://github.com/Yowkees/keyball/assets/142245946/1e030cb7-6c3f-47ec-854c-155a1ab9e6ce)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

12Pピンヘッダはヒートブロワー等で12ピンともに半田を溶かすことができると取り外せますが、半田ごてでは難しいです。
私の場合、無傷で外すのは難しいので、ピンソケットは壊してしまうつもりで、１～２ピンずつ半田を溶かして引き抜きます。

白銀ラボウェブサイトのお問い合わせよりご連絡いただければ、新しい12Pピンヘッダ単体での販売も可能です。

お手数ですがご対応お願いいたします。

**Comment by Piroshki01** [comment]
回答ありがとうございました。

ピンヘッダは犠牲になる覚悟で基盤外側はグラインダーで切断しました。
内側は切る術を持ち合わせてなかったので、内側のピンを軸にしてPromicroを傷つけなきように外側から持ち上げ、ピンを疲労破壊させて外しました。
Promicroを買ったお店で元々コンスルーも買っていたので、片側だけコンスルーにしてとりあえず使ってみようと思います。

今回のことはとても教訓になりました。
素晴らしいキットをありがとうございました。

**Comment by Yowkees** [comment]
工夫して取り外していただきありがとうございました！
ぜひKeyballの使い心地を楽しんでいただければ幸いです。
今後ともどうぞよろしくお願いいたします。
